src/app/pages/journal/journal.page.html

```diff
@@ -45,4 +45,8 @@
     <ng-template #slotContainer></ng-template>
   </ion-list>
 
+  <div *ngIf="displayNoDataMessage" class="no-journal-data">
```

Comment:
should be using the standard '.no-results' class,  which also uses a <p> tag instead of a <h3> tag. View Journal Card / Candidate Search card can be used as reference

Reply:
ok will change

Reply:
updated to use that class and use a p tag, but still needed an additional style to make the text centralised.

Reply:
https://ionicframework.com/docs/layout/css-utilities

Looks like we can use the built in ionic class instead of custom here i.e. class="ion-text-center"